README.md

```diff
@@ -54,7 +54,8 @@ $ make
 ```sh
 $ git clone https://github.com/yonaskolb/Mint.git
 $ cd Mint
-$ swift run mint install yonaskolb/mint --global
+$ swift run mint install yonaskolb/mint --
```

Comment:
Was this an intended change?

Reply:
I guess this was trying to remove the global flag, but missed the —
I’ve updated this in master

Reply:
ah, sorry, did not see the second part in the GitHub mail. This was indeed not intended, sry.